examples/common/python/connectors/direct/work_order_jrpc_impl.py

```diff
@@ -202,3 +191,19 @@ def work_order_get_result(self, work_order_id, id=None):
         }
         response = self.__uri_client._postmsg(json.dumps(json_rpc_request))
         return response
+
+    def work_order_get_result(self, work_order_id, id=None):
+        """
+        Get the work order result in blocking way until it get the result/error
+        It return json rpc response of dictionary type
+        """
+        response = self.work_order_get_result_nonblocking(work_order_id, id)
+        if "error" in response:
+            if response["error"]["code"] != 5:
+                return response
+            else:
+                while "error" in response and \
+                    response["error"]["code"] == 5:
+                    response = self.work_order_get_result_nonblocking(work_order_id, id)
+                    time.sleep(2)
```

Comment:
Add a comment "check for response after every 2 sec". TODO: add support for responseTimeoutMsecs